horiz_interp/horiz_interp_type.F90

```diff
@@ -117,8 +117,8 @@ subroutine stats ( dat, low, high, avg, miss, missing_value, mask )
  real, intent(in), optional :: missing_value
  real,    intent(in), optional :: mask(:,:)
 
- real :: dsum, npts, buffer_real(3)
- integer :: pe, root_pe, npes, p, buffer_int(2)
+ real :: dsum, buffer_real(3)
+ integer :: pe, root_pe, npes, p, buffer_int(2), npts
```

Comment:
Is there a chance npts can overflow as an integer here?

Reply:
Hard to say, we can keep it in mind if there's any issues with the tag.